polydatum/context.py

```diff
@@ -235,7 +232,11 @@ def __exit__(self, exc_type=None, exc_value=None, traceback=None):
                 if exc_type:
                     # An in-context or middleware exception
                     # occurred and will be raised outside the context
-                    six.reraise(exc_type, exc_value, traceback)
+                    if exc_value is None:
```

Comment:
This is basically the same thing `six.reraise` is doing. Could probably do just `raise exc_value` but I wanted to be safe.